src/renderers/action-parse.renderer.ts

```diff
@@ -105,12 +107,14 @@ export class ActionParseRenderer extends Renderer {
       return redirect
     })
 
-    return { data: data.parse.text, moduleDependencies, redirects: normalizedRedirects, displayTitle: data.parse.displaytitle }
+    const bodyCssClass = extractBodyCssClass(data.parse.headhtml)
```

Comment:
While you are at it, also get the classes for `<html>`. Vector-2022 has a lot of clientpref classes there.